js/app/cart/Cart.jsx

```diff
@@ -106,6 +118,10 @@ class Cart extends React.Component
     }).catch((err) => { console.log(err) });
   }
 
+  removeSessionTopCart() {
+    document.querySelector('a.btn-default').remove();
```

Comment:
- the method name is misleading as you are not removing the cart in session, you are just removing an UI element. The session cart will be updated with the new restaurant ID when adding a new item.
- the selector used to get the element is too broad. Please give an ID to the element you want do remove and use `document.getElementbyId('<my_id>').remove()`